### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by aron-net**
## [Project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md)

>  Add RSpec as a gem to the project.
>  Create a Request spec file for all the controllers.
>  Make sure that for each action you check:
 :heavy_check_mark: If the response status was correct. 
 :heavy_check_mark: If a correct template was rendered.
:heavy_check_mark: If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    end
+  end
+
+  describe 'GET /show' do
+    before(:each) { get user_path(1) }
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+    it "renders 'show' template" do
+      expect(response).to render_template('show')
```
Comment (FrederickMih) [inline spec/requests/posts_spec.rb]: - [ ] Hi @aron-helu, some of your test cases are failing. Let's kindly fix them.


Screenshot from 2022-11-01 19-01-24

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @aron-helu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Correct Gitflow has been followed :heavy_check_mark: 
- Descriptive README.md file
- Pull Request has a title and summary :heavy_check_mark: 
- No Linter error :heavy_check_mark: 


## Required Changes ♻️




_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb (hunk 1 above)
Comment (aron-net) [reply spec/requests/posts_spec.rb]: Already implemented it! Thanks for the feedback! :smile:

**Review by jaflih — APPROVED** [review]
Hi @aron-helu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Correct Gitflow has been followed ✔️
Descriptive README.md file ✔️
Pull Request has a title and summary ✔️
No Linter error ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**